src/api/docs/content/specs/domains.yaml

```
@@ -34,6 +34,16 @@ components:
           - `/api/domains/exact/abc.com` (allowed and denied exact domains identical to `abc.com`)
           - `/api/domains/regex` (allowed and denied regex domains)
           - `/api/domains/regex/abc.com` (allowed and denied regex domains identical to `abc.com`)
+        parameters:
+          - allOf:
+            - $ref: '#/components/parameters/type'
+            - required: false
+          - allOf:
+            - $ref: '#/components/parameters/kind'
+            - required: false
+          - allOf:
+            - $ref: '#/components/parameters/domain'
+            - required: false
```

```suggestion
        parameters:
          - allOf:
            - $ref: 'domains.yaml#/components/parameters/type'
            - required: false
          - allOf:
            - $ref: 'domains.yaml#/components/parameters/kind'
            - required: false
          - allOf:
            - $ref: 'domains.yaml#/components/parameters/domain'
            - required: false
```

Unfortunately, the tool we use for parsing and validating the specs against the real API in real tests on the CI does not work all that well on relative parts. My suggested changes above should make the CI tests pass.